modules/project-docs/pages/sdk-release-notes.adoc

```diff
@@ -28,11 +28,68 @@ Java 11 has various enhancements like lambda local variable type inference, prof
     <dependency>
         <groupId>com.couchbase.client</groupId>
         <artifactId>java-client</artifactId>
-        <version>3.0.4</version>
+        <version>3.0.5</version>
     </dependency>
 </dependencies>
 ----
 
+== Version 3.0.5 (2 June 2020)
+
+Version 3.0.5 is the sixth release of the 3.0 series, bringing enhancements and bugfixes over the last stable release.
+
+http://packages.couchbase.com/clients/java/3.0.5/Couchbase-Java-Client-3.0.5.zip[Download] |
+http://docs.couchbase.com/sdk-api/couchbase-java-client-3.0.5/[API Reference] |
+http://docs.couchbase.com/sdk-api/couchbase-core-io-2.0.6/[Core API Reference]
+
+The supported and tested dependencies for this release are:
+
+* com.couchbase.client:**java-client:3.0.5**
+* com.couchbase.client:**core-io:2.0.6**
+* io.projectreactor:**reactor-core:3.3.5.RELEASE**
+* org.reactivestreams:**reactive-streams:1.0.3**
+
+=== Enhancements
+
+* http://issues.couchbase.com/browse/JVMCBC-852[JVMCBC-852]:
+Bumped Reactor to 3.3.5, Netty to 4.1.50.Final, and Jackson to 2.11.0
+* http://issues.couchbase.com/browse/JCBC-1637[JCBC-1637]:
```

Comment:
I don't think this is release-note-worthy since it is mainly internal - people would get only confused by this

Reply:
This is just JCBC-1637 right? I'll remove it.

Reply:
yeah